examples/Demo/FluentUI.Demo.Client/Documentation/Components/Checkbox/Examples/CheckboxSizes.razor

```diff
@@ -0,0 +1,21 @@
+﻿<div>
```

Comment:
I don't think this example is relevant to the documentation.
Personally, I would remove the ‘Size’ example

Reply:
But I would add a "default" example in the top of the page:

```
<FluentStack Orientation="Orientation.Vertical">
    <FluentCheckbox @bind-Value="@value1">Apples</FluentCheckbox>
    <FluentCheckbox @bind-Value="@value2" Disabled="true">Bananas (disabled)</FluentCheckbox>
    <FluentCheckbox @bind-Value="@value3">Oranges</FluentCheckbox>
</FluentStack>

@code {
    bool value1 = true;
    bool value2 = true;
    bool value3;
}
```